flump/__init__.py

```diff
@@ -11,7 +11,7 @@
     :copyright: (c) 2015 by RolePoint.
 """
 
-import logging
+import logging as _logging
```

Comment:
clashed with my new kwargs. But I guess now I'm popping it from kwargs I can call it whatever.